### PR TITLE
Update free-programming-books-hi.md

### PR DESCRIPTION
**Pull request by huzefaTaj**
Adding a Python in Hindi

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 * [C++](#cpp)
+* [Python](#py)
 
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
```
Comment (Abinashbunty) [inline books/free-programming-books-hi.md:12]: 1 more blank line to be inserted.

File: books/free-programming-books-hi.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+### <a id="py"></a>Python
+* [Python Introduction \| Hindi](https://www.tutorialinhindi.com/python/history-of-python-hindi/) 
+* [Python DataTypes \| Hindi](https://www.tutorialinhindi.com/python/python-data-types-hindi/)
```
Comment (Abinashbunty) [inline books/free-programming-books-hi.md:15]: Sort these in alphabetical order.

**Review by Abinashbunty — CHANGES_REQUESTED** [review]

**Comment by shrn01** [comment]
Please fix the lint errors as specified by @Abinashbunty

**Comment by eshellman** [comment]
these are 3 pages of a website, not really a book. also intrusive advertising

**Pull request closed without merging**